tenzir/integration/data/reference/python/test_python_operator_empty_input/step_00.ref

```diff
@@ -0,0 +1 @@
+{"version": "4.11.2+gf8c6f3452d-dirty", "build": "gf8c6f3452d-dirty", "major": 4, "minor": 11, "patch": 2}
```

Comment:
Please use something other than `version` for testing.

Reply:
Ha, that's a _very_ good point. Thank you for catching this. 

(Kept `version`, but added `put x=42` afterwards.)